packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -106,14 +95,14 @@ class _VectorGraphicsWidgetState extends State<VectorGraphic> {
   PictureInfo? _pictureInfo;
 
   @override
-  void initState() {
+  void didChangeDependencies() {
```

Comment:
Do we need to do this every time - or just when the asset bundle changes?

Reply:
Oh - i see it could be used by the loader. maybe the loader should tell us if it needs to be re-downloaded

Reply:
I think if you want a caching loader you'll just have to write one.